neural_speed/models/CMakeLists.txt

```diff
@@ -13,6 +13,14 @@
 #  limitations under the License.
 
 file(GLOB MODEL_UTILS_SOURCE "model_utils/*.cpp")
+
+function(add_model)
+  add_library_w_warning(${TARGET} ${ModelSrcs})  # no (gpt) model utils needed
```

Comment:
shall ```ModelSrcs``` passed as variables instead of assuming it will always be defined otherwhere.

Reply:
changed in 8b071198bc8e319126b544763ce03bbb4b2a5fc0